tools/rna_tools/pipmir/tool-data/twobit_update.loc.sample

```diff
@@ -0,0 +1,23 @@
+#This is a sample file distributed with Galaxy that is used by some
```

Comment:
But you are right, there is something fishy ... I'm pretty sure this is working, so maybe you want to copy this: https://github.com/fidelram/deepTools/blob/fd4e8dc896c2ac6f838e35e8160046d6ac862692/galaxy/wrapper/deepTools_macros.xml#L530